src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -739,7 +739,6 @@ static Compiler()
             s_builtinAttributeGenerator.Add(typeof(ParameterAttribute), NewParameterAttribute);
             s_builtinAttributeGenerator.Add(typeof(OutputTypeAttribute), NewOutputTypeAttribute);
             s_builtinAttributeGenerator.Add(typeof(AliasAttribute), NewAliasAttribute);
-            s_builtinAttributeGenerator.Add(typeof(ValidateSetAttribute), NewValidateSetAttribute);
```

Comment:
I temporary removed that to pass tests but I forgot to ask you about that.
Now fixed.